### DISCUSSION TITLE
Spacing

### DISCUSSION THREAD

**Discussion by huandrew02** [microsoft/vscode-cpptools#7342] [Q&A] [answered]
So I reset my pc and redownloaded VSC and got the the extension. Before when I would write code,
eg. for(int i=1;i<=10;i++){
}
it would auto space after I move to a new line or after I enter " ; " or " { ". 
so, for(int i=1;i<=10;i++){
}
would become, for(int i = 1; i <= 10; i++){
} 

Im not 100% sure if this was the extension that did that but if it is what's the setting?

**Comment by sean-mcmanus** [comment, marked as answer]
That will happen if 
```
    "C_Cpp.formatting": "vcFormat",
    "editor.formatOnType": true
```
is used so you could change either of those to something else, e.g. "clangFormat" and/or "false". That's assuming the C/C++ extension did the formatting.
